packages/create-vite/template-lit-ts/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     /* Bundler mode */
     "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
-    "isolatedModules": true,
+    "verbatimModuleSyntax": true,
+    "erasableSyntaxOnly": true,
```

Comment:
I think this should be in the "Linting" section rather than "Bundler mode" section as it's not required for Vite. Maybe also good to add a comment that it can be disabled if desired.